code/testsuite/csheets/pf_Cleric.xml

```diff
@@ -1458,14 +1432,14 @@ BIO
 				<short>M</short>
 			</size>
 			<special_properties>Cures 1d8+1 points of damage</special_properties>
-			<type>MAGIC.POTION.CONSUMABLE</type>
+			<type>MAGIC.POTION.CONSUMABLE.ENHANCEMENT</type>
```

Comment:
Are one time recovery items really enhancements, since they have no lasting effect?

Reply:
That gets added by the EQMOD "Special Ability ~ Spell Effect / Single Use".